rem/remApp.swift

```diff
@@ -627,15 +630,19 @@ class AppDelegate: NSObject, NSApplicationDelegate {
                 backing: .buffered, defer: false)
             searchViewWindow?.hasShadow = false
             searchViewWindow?.ignoresMouseEvents = false
-
+            
             searchViewWindow?.center()
             searchViewWindow?.contentView = NSHostingView(rootView: searchView)
             
             searchViewWindow?.makeKeyAndOrderFront(nil)
-            searchViewWindow?.orderFrontRegardless() // Ensure it comes to the front
+            DispatchQueue.main.async {
+                self.searchViewWindow?.orderFrontRegardless() // Ensure it comes to the front
+            }
```

Comment:
someone explain main thread / ui thread in relation to dispatchqueue to me please